pulseapi/users/serializers.py

```diff
@@ -1,7 +1,24 @@
 """Serialize the models"""
 from rest_framework import serializers
 
-from pulseapi.users.models import EmailUser
+from pulseapi.users.models import (
+    EmailUser,
+    UserFavorites,
+)
+
+
+class UserFavoritesSerializer(serializers.ModelSerializer):
+    """
+    Serializes a {user,entry,when} fave.
+    """
+
+    user = '__all__'
```

Comment:
good question @gideonthomas do you remember why we needed this? If so, we can document it in the source